### PR TITLE
Sprint3 james - no post in 24 hours notification

### PR DESCRIPTION
**Pull request by perpetual-pelican**
Simple code for message display when user hasn't posted for 24 hours.

### CODE REVIEW

File: mhapsite/mhap/views.py [hunk 1]
```diff
@@ -207,16 +205,25 @@ def index(request):
     happy_graph = BarChart(data_source, options=options)
   
     instance = queryset.first()
-
-    second_quote = Quote.objects.get(id=2)
+    if instance:
+        current_time = datetime.datetime.utcnow().replace(tzinfo=utc)
```
Comment (akohli96) [inline mhapsite/mhap/views.py:209]: Looks good.

**Review by akohli96 — APPROVED** [review]

**Comment by akohli96** [comment]
James, this has more code then just current time and date. We should figure out how to only 'pull in' certain commits. 
Anyway @markiragoldberg when you reviewd it please merge it into master

File: mhapsite/mhap/admin.py [hunk 2]
```diff
@@ -7,8 +7,9 @@
 # Register your models here.
 from .models import Profile
 from .models import Post
-from .models import Quote
+from .models import Quote,ChatMessages
```
Comment (markiragoldberg) [inline mhapsite/mhap/admin.py:10]: This is inconsistent style, bro

File: mhapsite/mhap/bot_helper.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import re
```
Comment (markiragoldberg) [inline mhapsite/mhap/bot_helper.py:1]: Why do we import re here, but not use regular expressions?

File: mhapsite/templates/bot.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<h4> Welcome to the bot page </h4>
+
+<h4> {{data}}
+    <h4>
```
Comment (markiragoldberg) [inline mhapsite/templates/bot.html:4]: Think you meant to close the h4 here, not open another?

File: mhapsite/mhap/views.py [hunk 5]
```diff
@@ -305,3 +312,30 @@ def change_password(request):
...
+
+    #IF user is posted data
+    if form.is_valid():
+        print form.cleaned_data
+        info = form.cleaned_data['chat']
+        user_prof = Profile.objects.get(user=request.user)
+        print info
+        new_message = ChatMessages.objects.create(message=info,user_id=user_prof,is_user=True)
+        print new_message,"NEW MESSGE"
+        print info
```
Comment (markiragoldberg) [inline mhapsite/mhap/views.py:334]: Are all these print statements necessary in master?

**Comment by akohli96** [comment]
@markiragoldberg Another issue is that some of that code is mine not James since he pulled my changes in to merge to master.
@pelikenesis please look if we can only merge certain commits. We should leave my code out of master for time being.

**Comment by markiragoldberg** [comment]
I have no issues with this PR beyond the token issues I've raised, but if we need to separate out Ayush's code then so be it.

**Comment by perpetual-pelican** [comment]
@akohli96 Yeah, I didn't think about that. I guess that is an option though, so I'll remember that in the future.

**Comment by perpetual-pelican** [comment]
@markiragoldberg We could just drop this pull request and have @akohli96 clean up a couple things and merge his changes first

**Comment by akohli96** [comment]
@pelikenesis My stuff is under dev and since Chat is not ready, I dont see why we should merge that.
Please use [git cherry-pick](http://stackoverflow.com/questions/881092/how-to-merge-a-specific-commit-in-git) and add your requests for reviews.

Also, please close this pull request

**Comment by perpetual-pelican** [comment]
Okay, I'll check it out tonight. Closing this pull request now.

**Pull request closed without merging**